virtual_workstation_setup.sh

```diff
@@ -170,32 +170,29 @@ setup_master() {
     echo "Finished setting up the mongo repo..."
 }
 
-setup_50() {
+setup_60() {
```

Comment:
Should we not *also* be setting up 5.0? I think the goal here was to have all current branches set up.

Reply:
Then we should setup v5, v4.4, and v4.2 but we didn't do that before. I am just keeping things consistent.